test/basic/array2.c

```diff
@@ -3,7 +3,7 @@
 #include "smack.h"
 
 // @flag --loop-limit=11
-// @flag --unroll=11
+// @flag --unroll=100
```

Comment:
Ah, I see what is going on here. I would prefer a different temporary solution. You should leave these unrolls as they were. Then in the `top.py` file you should add a hacky solution where you multiply the unroll factor by e.g. 10 when being invoked with Symbooglix. I hope this makes sense.